docs_source_files/content/introduction/types_of_testing.en.md

```diff
@@ -15,8 +19,13 @@ Note: Acceptance testing is a subtype of **_functional testing_**,
 which some people might also refer to.
             
 ### Functional testing
-This type of test is done to determine if a product's
-feature functions properly, without issues.
+This type of testing is done to determine if a
+feature or system functions properly without issues. It checks
+the system at different levels to ensure that all scenarios
+are covered and that the system does _what's_ 
+supposed to do. It's a verification activity that
+answers the question:
+>Are we building the product **_right?_**.
```

Comment:
hi @ivanrodjr ,

This is great, also can you please make changes in other translated pages too. If already translated, you can replace the text with plain English, and we can translate them again.

Regards,
Harsha

Reply:
Sure, no problem

Reply:
@Harsha509 done!